Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -3912,6 +4136,256 @@ def screenshot_url_command(client: Client, args: dict[str, Any]) -> CommandResul
+    """
+    feed_uuid = args.get("feed_uuid")
+    if not feed_uuid:
+        raise ValueError("Feed UUID is required")
+
+    indicators = args.get("indicators")
+    if not indicators:
+        raise ValueError("Indicators is required")
+
+    result = client.add_indicators(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.AddIndicators",
+        outputs_key_field="feed_uuid",
+        outputs=result,
+        readable_output=tableToMarkdown("SilentPush Add Indicators", result),
+        raw_response=result,
+    )
+
+
+@metadata_collector.command(
+    command_name="silentpush-add-indicator-tags",
+    inputs_list=ADD_INDICATOR_TAGS_INPUTS,
+    outputs_prefix="SilentPush.AddIndicatorTags",
+    outputs_list=ADD_INDICATOR_TAGS_OUTPUTS,
+    description="This command updates tags to the indicators",
+)
+def add_indicators_tags_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler for add new indicator tags.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'feed_uuid' and 'indicator_name key.
+
+    Returns:
+        CommandResults: JSON response of added indicator tags.
+    """
+    feed_uuid = args.get("feed_uuid")
+    if not feed_uuid:
+        raise ValueError("Feed UUID is required")
+
+    indicator_name = args.get("indicator_name")
+    if not indicator_name:
+        raise ValueError("Indicator name is required")
+
+    if not args.get("tags"):
+        raise ValueError("The 'tags' argument is required and cannot be empty.")
+
+    result = client.add_indicators_tags(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.AddIndicatorTags",
+        outputs_key_field="feed_uuid",
+        outputs=result,
+        readable_output=tableToMarkdown("SilentPush Add Indicator Tags", result),
+        raw_response=result,
+    )
+
+
+@metadata_collector.command(
+    command_name="silentpush-run-threat-check",
+    inputs_list=RUN_THREAT_CHECK_INPUTS,
+    outputs_prefix="SilentPush.RunThreatCheck",
+    outputs_list=RUN_THREAT_CHECK_OUTPUTS,
+    description="This command runs the threat check on the specified ",
+)
+def run_threat_check_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler to fetch threat checks.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'feed_uuid' key.
+
+    Returns:
+        CommandResults: JSON response of threat check.
+    """
+    threat_type = args.get("type")
+    if not threat_type:
+        raise ValueError("Threat type is required")
+
+    data = args.get("data")
+    if not data:
+        raise ValueError("Data is required")
+
+    user_identifier = args.get("user_identifier")
+    if not user_identifier:
+        raise ValueError("User identifier is required")
```

Comment:
if the arg is set to required in the yml, no need for this check, the command will not run without it.